dask/dataframe/shuffle.py

```diff
@@ -645,7 +670,13 @@ def set_sorted_index(df, index, drop=True, divisions=None, **kwargs):
     result = map_partitions(M.set_index, df, index, drop=drop, meta=meta)
 
     if not divisions:
-        divisions = compute_divisions(result, **kwargs)
+        divisions, flag = compute_divisions(result, **kwargs)
```

Comment:
It would be nice to find a more descriptive variable name for `flag`.  I think that this is something like *overlapping_divisions* (but probably a better name exists)

Reply:
Agree. Thanks.